src/org/traccar/protocol/AustinNbProtocolDecoder.java

```diff
@@ -57,11 +63,16 @@ protected Object decode(
         Position position = new Position(getProtocolName());
         position.setDeviceId(deviceSession.getDeviceId());
 
-        position.setTime(parser.nextDateTime());
+        position.setTime(parser.nextDateTime(Parser.DateTimeFormat.YMD_HMS, TimeZone.getDefault().getID()));
 
         position.setValid(true);
-        position.setLongitude(Double.parseDouble(parser.next().replace(',', '.')));
         position.setLatitude(Double.parseDouble(parser.next().replace(',', '.')));
+        position.setLongitude(Double.parseDouble(parser.next().replace(',', '.')));
+        position.set("azimuth", parser.nextInt());
+        position.set("angle", parser.nextInt());
+        position.set("range", parser.nextInt());
+        position.set("out_of_range", parser.nextInt());
```

Comment:
This doesn't comply with naming convention. Also, what does it mean?

Reply:
An extra info we need for the protocol.
I will name by the convention.